massa-node/src/main.rs

```diff
@@ -80,13 +80,11 @@ use massa_protocol_exports::{ProtocolConfig, ProtocolManager};
 use massa_protocol_worker::{create_protocol_controller, start_protocol_controller};
 use massa_storage::Storage;
 use massa_time::MassaTime;
-use massa_versioning::versioning::{
-    ComponentStateTypeId, MipComponent, MipInfo, MipState, MipStatsConfig, MipStore, StateAtError,
-};
+use massa_versioning::versioning::{ComponentStateTypeId, MipStatsConfig, MipStore, StateAtError};
```

Comment:
Idk how we're planning on deploying nodes with MIPs but removing these dependencies means we'll have to remember to put it in the deploy script if that's the way we go.

Reply:
I've moved the MIP info declaration in versioning module + unused imports.